model/proxy.yaml

```
@@ -0,0 +1,14 @@
+groups:
+  - id: proxy
+    prefix: proxy
+    type: attribute_group
+    brief: These attributes may be used to describe a known proxy (or other intermediary) in a connection-based network interaction.
+    attributes:
+      - id: address
+        type: string
+        brief: Proxy address - domain name if available without reverse DNS lookup, otherwise IP address or Unix domain socket name.
+        examples: ['proxy.example.com', '10.1.2.80', '/tmp/my.sock']
```

nit:
```suggestion
        examples: ['proxy.example.com', '10.1.2.80']
```

not sure if the unix domain socket proxy exists :)